node-js-basics-101/node_modules/asynquence-contrib/contrib-common.js

```diff
@@ -0,0 +1,5 @@
.apply(t,arguments)}),r}function addWrapped(){c.push.apply(c,r.call(arguments).map(wrap))}function iterateOrQuit(r,u){c.length>0?u?r():a(r):("undefined"!=typeof f?e.isMessageWrapper(f)||(f=[f]):f=[],n.apply(t,f))}var s,o,l,c=i,p={messages:r.call(arguments,1),add:addWrapped},f=p;c=c.map(wrap),function iterate(r){o=r?"throw":"next",s=c.shift();try{l=e.isMessageWrapper(f)&&e.isSequence(s)?s[o].apply(s,f):s[o](f)}catch(a){return n.fail(a)}if(!u("seq_aborted"))if(l.value===p)l.done||c.push(s),f=p,iterateOrQuit(iterate,!1);else{if(!e.isSequence(l.value))if(isPromise(l.value))l.value=e().promise(l.value);else if("function"==typeof l.value){var i=l.value;l.value=e(function $$ASQ(e){i(e.errfcb)})}else e.isMessageWrapper(l.value)?l.value=e.apply(t,l.value.length>0?l.value:e.messages(void 0)):"undefined"!=typeof l.value?l.value=e(l.value):l.value=e();l.value.val(function $$val(){u("seq_aborted")||(arguments.length>0&&(f=arguments.length>1?e.messages.apply(t,arguments):arguments[0]),l.done||(f===p?c.push(s):c.unshift(s)),iterateOrQuit(iterate,!0))}).or(function $$or(){u("seq_aborted")||(l.done?n.fail.apply(t,arguments):(c.unshift(s),f=arguments.length>1?e.messages.apply(t,arguments):arguments[0],iterate(!0)))})}}()}),n}}),e.extend("toPromise",function $$extend(n,a){return function $$to$promise(){return new Promise(function $$executor(a,u){n.val(function $$val(){var n=r.call(arguments);return a.call(t,n.length>1?n:n[0]),e.messages.apply(t,n)}).or(function $$or(){var e=r.call(arguments);u.call(t,e.length>1?e:e[0])})})}}),e.wrap=function $$wrap(n,a){function checkThis(e,r){return!e||"undefined"!=typeof window&&e===window||"undefined"!=typeof global&&e===global?r:e}function paramSpread(e){return regeneratorRuntime.mark(function paramSpread(r){return regeneratorRuntime.wrap(function paramSpread$(t){for(;;)switch(t.prev=t.next){case 0:return t.delegateYield(e.apply(this,r.messages),"t0",1);case 1:case"end":return t.stop()}},paramSpread,this)})}var u,i,s,o,l;if(a=a&&"object"==typeof a?a:{},a.errfcb&&a.splitcb||a.errfcb&&a.simplecb||a.splitcb&&a.simplecb||"errfcb"in a&&!a.errfcb&&!a.splitcb&&!a.simplecb||a.params_first&&a.params_last||a.spread&&!a.gen)throw Error("Invalid options");return o=a["this"]&&"object"==typeof a["this"]?a["this"]:t,u=a.errfcb||!(a.splitcb||a.simplecb),i=!!a.params_first||!a.params_last&&!("params_first"in a||a.params_first)||"params_last"in a&&!a.params_first&&!a.params_last,l=!(!a.spread&&"spread"in a),s=i?"push":"unshift",a.gen?(l&&(n=paramSpread(n)),function $$wrapped$gen(){return e(e.messages.apply(t,arguments)).runner(n)}):u?function $$wrapped$errfcb(){var t=r.call(arguments),a=checkThis(this,o);return e(function $$asq(e){t[s](e.errfcb),n.apply(a,t)})}:a.splitcb?function $$wrapped$splitcb(){var t=r.call(arguments),a=checkThis(this,o);return e(function $$asq(e){t[s](e,e.fail),n.apply(a,t)})}:a.simplecb?function $$wrapped$simplecb(){var t=r.call(arguments),a=checkThis(this,o);return e(function $$asq(e){t[s](e),n.apply(a,t)})}:void 0},e});
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Expected a conditional expression and instead saw an assignment.

<!-- Comment by [SideCI](https://sideci.com) -->